src/Microsoft.Graph/Models/Generated/ChatMessageMention.cs

```diff
@@ -19,7 +19,8 @@ namespace Microsoft.Graph
     /// The type ChatMessageMention.
     /// </summary>
     [JsonObject(MemberSerialization = MemberSerialization.OptIn)]
-    public partial class ChatMessageMention : Entity
+    [JsonConverter(typeof(DerivedTypeConverter))]
+    public partial class ChatMessageMention
```

Comment:
Haven't we already dropped Entity last week? Interesting..

Reply:
Oh, it was beta. Never mind.

Reply:
Still worth noting that this made its way into production in v1 from that docs bug.

Reply:
Yeah, I realized that last week after I fixed the bug. I don't expect to see much issue from of it.